src/mmw/js/src/modeling/templates/controls/summary.html

```diff
@@ -0,0 +1,3 @@
+<h2 class="dark">{{label}}</h2>
+<br>
+<span>{{summary}}</span>
```

Comment:
From a semantic HTML point of view I don't like using <br> tags for spacing. They're really for things like poems:

```html
<p>Sing in me, Muse, and through me tell the story<br>
of that man skilled in all ways of contending,<br>
the wanderer, harried for years on end,<br>
after he plundered the stronghold<br>
on the proud height of Troy.</p>